### PR TITLE
add two tower

### PR DESCRIPTION
**Pull request by shenzhimo2**
If you are opening a PR for `Notebooks` under the [official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Notebooks` under the [community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.

If you are opening a PR for `Community Content`, and it will NOT be used on cloud.google.com/docs:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.

If you are opening a PR for `Community Content`, and it will be used on cloud.google.com/docs:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb) for all the notebooks in your content directory.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify each notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the content has been reviewed by a tech writer, and they have approved it.
- [ ] This content has been added (/moved) to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Official Community Content` section (/from the `# Community Content` section), pointing to the author or the author's team.
- [ ] Each notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by shenzhimo2** [comment]
@googlebot I fixed it.

File: notebooks/official/matching_engine/two-tower-model-introduction.ipynb
Comment (ivanmkc) [inline notebooks/official/matching_engine/two-tower-model-introduction.ipynb:1]: <p>You'll need to fix the links so that they point to this notebook (after it gets merged).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ivanmkc) [inline notebooks/official/matching_engine/two-tower-model-introduction.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    !gsutil ls {OUTPUT_DIR}
</pre><p>Add space after exclamation to be consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (ivanmkc) [inline notebooks/official/matching_engine/two-tower-model-introduction.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    !gcloud ai endpoints predict {ENDPOINT_ID} \
</pre><p>Space after exclamation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Comment (ivanmkc) [inline notebooks/official/matching_engine/two-tower-model-introduction.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #68.    !curl -X POST -H "Authorization: Bearer "$(gcloud auth print-access-token) \
</pre><p>Space after exclamation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

**Comment by ivanmkc** [comment]
@googlebot I fixed it.

**Comment by shenzhimo2** [comment]
@googlebot I fixed it.

**Comment by shenzhimo2** [comment]
@googlebot I fixed it.

**Review by ivanmkc — APPROVED** [review]
LGTM once CODEOWNERS is modified.

**Pull request merged**
